src/longident.ml

```diff
@@ -51,10 +51,14 @@ let parse s =
   let invalid () =
     invalid_arg (Printf.sprintf "Ppxlib.Longident.parse: %S" s)
   in
-  match (String.index_opt s '(', String.rindex_opt s ')') with
-  | None, None -> parse_simple s
-  | None, _ | _, None -> invalid ()
-  | Some l, Some r -> (
+  let open_par = String.index_opt s '(' in
+  let close_par = String.index_opt s ')' in
+  let first_char_is_normal = is_normal_ident_char s.[0] in
```

Comment:
Since we have the `invalid` function, it makes sens to test whether `s` has length at least 1.

Reply:
Good point, just fixed it!